### PR TITLE
Create dnj-colab.yml

### PR DESCRIPTION
**Pull request by smaicas**
Create Dnj Colab project file. Educational Opensource Colaborative repository. Also linked to real-time coding streaming.

### CODE REVIEW

File: _data/projects/dnj-colab.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+tags:
+  - c#
+  - .net
+  - opensource
+  - education
+  - community
+  - streaming
+upforgrabs:
+  name: help wanted
+  link: https://github.com/smaicas-org/Dnj.Colab/issues
```
Comment (shiftkey) [inline _data/projects/dnj-colab.yml:13]: :art: this link should be to a label in the repository:

```suggestion
  link: https://github.com/smaicas-org/Dnj.Colab/labels/help%20wanted
```

File: _data/projects/dnj-colab.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Dev N´ Joy Colab 
+desc: Educational Opensource Colaborative repository to improve and learn programming. Samples, Training, Projecs...and real-time coding streaming!
```
Comment (shiftkey) [inline _data/projects/dnj-colab.yml:2]:
```suggestion
desc: Educational Opensource Colaborative repository to improve and learn programming. Samples, Training, Projects...and real-time coding streaming!
```

**Review by shiftkey — COMMENTED** [review]
There are no open issues in the repository, but otherwise this is good to proceed

**Comment by Erantr** [comment]
מי זה

בתאריך יום ו׳, 10 בפבר׳ 2023, 19:21, מאת Brendan Forster ‏<
***@***.***>:

**Comment by ritwik12** [comment]
Closing due to PR being inactive from a long time.

**Pull request closed without merging**